src/main/java/com/exasol/errorcodecrawlermavenplugin/ErrorValidator.java

```diff
@@ -0,0 +1,51 @@
+package com.exasol.errorcodecrawlermavenplugin;
+
+import java.io.File;
+import java.util.*;
+
+import com.exasol.errorcodecrawlermavenplugin.model.ExasolError;
+import com.exasol.errorreporting.ExaError;
+
+/**
+ * Validator for {@link ExasolError}s.
+ */
+public class ErrorValidator {
```

Comment:
can we rename it to `ExasolErrorValidator ` or `ExasolErrorInvocationValidator` ?